CHANGELOG.md

```diff
@@ -7,8 +7,12 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 ### Added
 - Add in-memory docker image
 - Support of AWS S3 as blobstore
+- WebAdmin routes for CLI capabilities on mapping management (AddAddressMapping, AddRegexMapping, ListMappings)
+- Metrics for Deleted Message Vault
+- More metrics for BlobStore (new delete & deleteBucket configuration)
 
 ### Changed
+- (experimental) Native implementation of the deleted message vault on top of an object store. This avoids exposing messages via webAdmin. Messages previously stored in the vault will be lost.
```

Comment:
does it worth putting in the changelog if it's experimental?

Reply:
IMO yes

Reply:
so we'll have to go to the maintenance/deprecation process in case of change, that looks annoying for few benefits

Reply:
Or we can remove the `(experimental)` from here...

Reply:
I do not understand, if it's unstable, either we do not make it public, or we flag it as being unstable.

Reply:
for me experimental is ok: you tell that there is a new feature, but also that it's experimental, so "use it at your very own risks"